### PR TITLE
F/support ssl rmq instance

### PR DESCRIPTION
**Pull request by ctpaterson**
Check options[:trust_manager] as Array, and only wrap if necessary.  Add support for options[:sasl_config] if set.

### CODE REVIEW

File: lib/march_hare/session.rb [hunk 1]
```diff
@@ -103,14 +103,16 @@ def self.connect(options = {})
             kmf.init(ks, pwd)
 
             if options[:trust_manager]
-              ctx.init(kmf.get_key_managers, [options[:trust_manager]], nil)
+              tm_a = options[:trust_manager].kind_of?(Array) ? options[:trust_manager] : [options[:trust_manager]]
```
Comment (michaelklishin) [inline lib/march_hare/session.rb]: There's `Kernel#Array` that does this.

Comment (ctpaterson) [reply lib/march_hare/session.rb]: Ah, so Array(options[:trust_manager])?  Much simpler.  I'll fix.

Comment (ctpaterson) [reply lib/march_hare/session.rb]: Done

**Comment by michaelklishin** [comment]
Thank you!

**Comment by ctpaterson** [comment]
Thank you, Mr. Klishin!  This was all from a fix we needed in our local use of MarchHare, but we took care to ensure they were changes we could send back.  It's gratifying to see them incorporated so quickly.

**Pull request merged**